### PR TITLE
Liquid layout

### PR DESCRIPTION
**Pull request by jtquach1**
## Describe your changes

> Please include a summary of the changes and the related issue/task. Please also include relevant motivation and context. List any dependencies that are required for this change, including links to other pull requests/branches in other repositories if applicable.

This PR implements WCAG techniques G146, G179, G199
* Liquid layout
* no loss of content or functionality when zooming in
* indication of form submission success (existing for submitting REMS bundle, but fixed bug with form loading)

It turns out LForms actually has built in display options for small, medium, and large screen sizes if one clicks the settings cog icon and selects an option from the View Mode dropdown. So that also already covers G146 and G179 on the QuestionnaireForm page.

[Default breakpoints in MUI 5](https://mui.com/material-ui/customization/breakpoints/#default-breakpoints) to reference for below

---

### xs screen  (problematic)
image
image

### sm screen (problematic at breakpoint, but otherwise OK)
image
image

image
image

### lg screen (OK)
image
image

### xs screen (after pre-wrapping whitespace, fixing ETASU/MedicationDispense.status text spacing, and text-wrapping MedicationRequest dropdown)
image
image

### sm screen (after fixing ETASU/MedicationDispense.status text spacing)
image

### md screen (after pre-wrapping whitespace)
image

---

### Applies to all screen sizes xs, sm, md, lg - horizontal scrollbar for FHIR Resource JSON inside `<pre>` tag
image

image

image

image

## After - Document Status
image

image

image

image

---

### xs screen (problematic cut-off text with input fields' helper text and values)
image

### xs screen (no padding)
image
image

### sm screen and above (original padding)
image
image

---

## Issue ticket number and Jira link

> Please include the Jira Ticket Number and Link for this issue/task.

https://jira.mitre.org/browse/REMS-654

## Checklist before requesting a review
- [ ] I have performed a self-review of my code
- [ ] Ensure the target / base branch for any feature PR is set to `dev` not main (the only exception to this is releases from `dev` and hotfix branches)

## Checklist for conducting a review
- [ ] Review the code changes and make sure they all make sense and are necessary. 
- [ ] Pull the PR branch locally and test by running through workflow and making sure everything works as it is supposed to. 

## Workflow 

Owner of the Pull Request will be responsible for merge after all requirements are met, including approval from at least one reviewer. Additional changes made after a review will dismiss any approvals and require re-review of the additional updates. Auto merging can be enabled below if additional changes are likely not to be needed. The bot will auto assign reviewers to your Pull Request for you.

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -3,18 +3,22 @@
   <head>
     <meta charset="utf-8" />
     <link rel="icon" href="%PUBLIC_URL%/favicon.ico" />
-    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.2.1/css/bootstrap.min.css"
-    integrity="sha384-GJzZqFGwb1QTTN6wy59ffF1BuGJpLSa9DkKMp0DgiMDm4iYMj70gZWKYbI706tWS" crossorigin="anonymous" />
-  <link href="https://clinicaltables.nlm.nih.gov/lforms-versions/29.0.3/styles/lforms.min.css" media="screen"
-    rel="stylesheet" />
-    <script src='https://kit.fontawesome.com/a076d05399.js'></script>
+    <link
```
Comment (jtquach1) [inline public/index.html:6]: prettier

File: src/views/Questionnaire/components/RemsInterface/RemsInterface.tsx [hunk 2]
```diff
@@ -36,42 +21,30 @@ export default function RemsInterface(props: RemsInterfaceProps) {
     return bundle.entry?.map(entry => {
```
Comment (zacharyrobin) [inline src/views/Questionnaire/components/RemsInterface/RemsInterface.tsx:21]: This seems to be causing a unique prop key warning. doesnt cause any breaking changes but may want to address this at some point 
Screenshot 2024-05-17 at 10 59 31 AM
Screenshot 2024-05-17 at 10 59 35 AM

**Review by zacharyrobin — COMMENTED** [review]
Merge Conflicts need to be addressed

**Review by KeeyanGhoreshi — CHANGES_REQUESTED** [review]
When you load a previous form, then modify it, then try to save it to the ehr, it reverts your form back to the previously loaded version, erasing any progress.

**Comment by jtquach1** [comment]
> When you load a previous form, then modify it, then try to save it to the ehr, it reverts your form back to the previously loaded version, erasing any progress.

Thanks for catching that, should be fixed now

**Review by KeeyanGhoreshi — APPROVED** [review]

**Pull request merged**